### PR TITLE
Lesson 18 Wiki Notes

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: Class-18-(ORM-Part-3).md
Comment (CarlyLanglois) [inline Class-18-(ORM-Part-3).md]: I would also say that with today's material, they have all they need to complete it.

File: Class-18-(ORM-Part-3).md
Comment (CarlyLanglois) [inline Class-18-(ORM-Part-3).md]: in preparation to discuss

File: Home.md
Comment (CarlyLanglois) [inline Home.md]: oops add this link!

**Review by CarlyLanglois — APPROVED** [review]
good to go after a merge with master and a few comments

File: Class-18-(ORM-Part-3).md
Comment (gildedgardenia) [reply Class-18-(ORM-Part-3).md]: Done!

File: Class-18-(ORM-Part-3).md
Comment (gildedgardenia) [reply Class-18-(ORM-Part-3).md]: Done!

File: Home.md
Comment (gildedgardenia) [reply Home.md]: Done!

**Pull request merged**
